Tzkt.Api/Websocket/Parameters/OperationsParameter.cs

```diff
@@ -44,6 +45,9 @@ public void EnsureValid()
 
             if (TypesList.Count == 0)
                 throw new HubException("Invalid operation types");
+
+            if (Entrypoints?.Count != 0 && !(TypesList.Count == 1 && TypesList[0] == Operations.Transactions))
+                throw new HubException("`Entrypoint` field can be used with `transactions` type only");
```

Comment:
No need to disallow subscribing to calls+other op types. For example, subscribing to `[ "xtzToToken", "tokenToXtz" ]` + all `origination`s would be helpful when indexing all dex trades, etc. So,
```cs
if (Entrypoints?.Count > 0 && !TypesList.Contains(Operations.Transactions))
    throw new HubException("`Entrypoints` can be used with `transactions` type only");
```

Reply:
Also, would be nice to disallow empty `Entrypoints` array:
```cs
if (Entrypoints?.Count == 0)
    throw new HubException("`Entrypoints` field can be either `null` or a non-empty array");
```